port: /dev/base_serial_port #ttyUSB0 # base_serial_port
baud: 921600 # 115200 
base_frame: base_link
odom_frame: odom
joy_topic:  /joy
odom_topic: /odom
imu_frame:  imu_frame
laser_frame:  laser_frame
base_shape_a: 0.2169
wheel_radius: 0.04657 
vel_topic: /cmd_vel
# provide_odom_tf: true
cmd_timeout: 0.2
serial_timeout: 50
linear_speed_max:  3.0
angular_speed_max: 3.14
Mileage_file_name:  car_Mileage_info.txt
debug_log:  false
